src/theme.c

```diff
@@ -1023,3 +1023,25 @@ theme_finish(struct theme *theme)
 	theme->corner_top_right_active_normal = NULL;
 	theme->corner_top_right_inactive_normal = NULL;
 }
+
+void theme_customize_with_border_color(struct theme *theme, float *color)
```

Comment:
This should be a static function in `ssd/ssd_titlebar.c`, where it's used.

Reply:
replaced with get_theme_for_view() as suggested